src/grpc/lightning.ts

```diff
@@ -171,8 +171,8 @@ export async function queryRoute(
       routes: [],
     }
   }
-  return new Promise(async function (resolve, reject) {
-    const lightning = await loadLightning(true, ownerPubkey) // try proxy
+  const lightning = await loadLightning(true, ownerPubkey) // try proxy
+  return new Promise((resolve, reject) => {
```

Comment:
I'm confused about what swapping these two lines does was it something eslint was throwing a warning about?

Also what does removing the async from the function do, does it affect functionality

Reply:
eslint complains about 'promise executor functions' being async, when removing the async there, await doesn't work anymore inside this function so i moved it a line, also somewhere an rpc was loaded two times inside the function and moving it up made it load only once

Reply:
ok cool that is a good improvement then

@Evanfeenstra let me know what you think